dpctl/apis/include/dpctl4pybind11.hpp

```diff
@@ -0,0 +1,100 @@
+//===----------- dpctl4pybind11.h - Headers for type pybind11 casters  -*-C-*-
+//===//
+//
+//                      Data Parallel Control (dpctl)
+//
+// Copyright 2020-2021 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+//===----------------------------------------------------------------------===//
+///
+/// \file
+/// This file defines imports for dcptl's Python C-API
+//===----------------------------------------------------------------------===//
+
+#pragma once
+
+#include "dpctl_capi.h"
+#include <CL/sycl.hpp>
+#include <pybind11/pybind11.h>
+
+namespace py = pybind11;
+
+namespace pybind11
+{
+namespace detail
+{
+template <> struct type_caster<sycl::queue>
```

Comment:
Please add docstrings. So that we can start generating these files in API docs under maybe programmer's guide.

Reply:
Done. Added comments to document what's going on.